### DISCUSSION TITLE
v1.6.1

### DISCUSSION THREAD

**Discussion by Spacesai1or** [ethereum/web3.js#4545] [Announcements]
### Added

- Support for `eth_createAccessList` as both an rpc call (`web3.eth.createAccessList`) and property of contract method wrappers (`contractInstance.methods.getValue().createAccessList`) (#4332)

### Changed

- Not considering `tx.chainId` if `tx.common.customChain.chainId` is provided for `web3.eth.accounts.signTransaction` function (#4293)
- Added missing PromiEvent handler types (#4194) 
- Updated README to include Webpack 5 angular support instructions (#4174)
- Updated the documentation for the `Web3.utils`, removed context for `_` (underscore lib) (#4403)
- Emit subscription id with connect event when creating a subscription (#4300)
- Introduced new configuration "blockHeaderTimeout" for waiting of block headers for transaction receipt  (#3891)
- Format `block.baseFeePerGas` to number (#4330)
- Correct `web3-eth-personal.sendTransaction` example in documentation (#4409)
- Updated README to include Webpack 5 angular support instructions (#4174)

### Fixed

- Fix 1.6.1 build size issue with removing static asset files (#4506)
- Correct `web3.rst` example in documentation (#4511)
- Correct `BlockHeader` typing (`receiptRoot` -> `receiptsRoot`) (#4452)

<hr /><em>This discussion was created from the release <a href='https://github.com/ChainSafe/web3.js/releases/tag/v1.6.1'>v1.6.1</a>.</em>

**Comment by Shoot-it** [comment]
thank you
